src/codeu/chat/util/ServerInfo.java

```diff
@@ -9,8 +9,6 @@ public ServerInfo(Time startTime) {
 		this.startTime = startTime;
 	}
 	public String toString(){
-		String out = "";
-		out = startTime.toString();
-		return out;
```

Comment:
I do believe I had to manually remove this from the merge. I really don't know why this simple operation would be done in such a protracted manner.